tools/bumpVersion.sh

```diff
@@ -7,38 +7,30 @@ readonly TOP
 
 readonly PACKAGE_JSON="$TOP/package.json"
 
-# Read the version from package.json as the "current version"
-CURRENT_VERSION="$(grep -E '"version": "([0-9\.]+)"' "$PACKAGE_JSON" | awk -F\" '{print $4}')"
+# Save the current package.json version to a variable
+CURRENT_VERSION="$(jq -r '.version' < "$PACKAGE_JSON")"
 readonly CURRENT_VERSION
 
 # Parse the version into major/minor/patch
-readonly MAJOR="${CURRENT_VERSION%%.*}"
+readonly _MAJOR="${CURRENT_VERSION%%.*}"
```

Comment:
This change was just to make shellcheck happy about this unused variable

Reply:
Oh ha you can just remove it